eng/Versions.props

```diff
@@ -10,7 +10,7 @@
     <MicrosoftDotNetBuildTasksTemplatingPackageVersion>10.0.0-beta.24604.4</MicrosoftDotNetBuildTasksTemplatingPackageVersion>
     <MicrosoftTestingExtensionsCodeCoverageVersion>17.13.1</MicrosoftTestingExtensionsCodeCoverageVersion>
     <!-- comment to facilitate merge conflicts -->
-    <MicrosoftTestingExtensionsRetryVersion>1.5.0-preview.24627.3</MicrosoftTestingExtensionsRetryVersion>
+    <MicrosoftTestingExtensionsRetryVersion>1.5.0-preview.24630.2</MicrosoftTestingExtensionsRetryVersion>
```

Comment:
@Evangelink Looks like both versions are actually not on NuGet.org and should be stable?

Reply:
Looking more on the usages of this property in that branch (it's no longer needed in main as we moved Retry to testfx), I'm so confused by the way MSTest.Sdk is doing it.

https://github.com/microsoft/testfx/blob/a75935df55a104d80e783f8bdde09a9e1f77de03/src/Package/MSTest.Sdk/Sdk/Runner/ClassicEngine.targets#L25-L27

The default version is set via `MicrosoftTestingEntrepriseExtensionsVersion`. But there is no way for that property to be defined in user apps.

It seems like the intent was for a "replacement" to happen via:

https://github.com/microsoft/testfx/blob/a75935df55a104d80e783f8bdde09a9e1f77de03/src/Package/MSTest.Sdk/MSTest.Sdk.csproj#L38

But we don't setup ClassicEngine.targets for such replacement.

Reply:
I'll explain offline